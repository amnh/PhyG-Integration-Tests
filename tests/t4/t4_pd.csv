Alentus,Amblypygid,Aportus,Centruoides,Hadrurus,Limulus,Mastigoproctus,Pauroctonus,Rhiphicephalus,Thelichoris,Thermobius,Vonones
"0.0","35.0","12.0","46.0","45.0","40.0","43.0","44.0","45.0","43.0","44.0","34.0"
"35.0","0.0","37.24503311258278","31.042253521126757","38.53521126760563","25.69014084507042","24.447552447552447","33.183098591549296","39.834482758620695","21.408450704225352","41.54666666666667","24.27777777777778"
"12.0","37.24503311258278","0.0","46.3046357615894","48.31788079470199","42.27814569536424","42.27814569536424","48.31788079470199","49.32450331125828","42.27814569536424","47.311258278145694","38.25165562913907"
"46.0","31.042253521126757","46.3046357615894","0.0","32.112676056338024","31.26241134751773","25.51048951048951","24.79432624113475","45.07586206896552","28.901408450704224","41.54666666666667","34.833333333333336"
"45.0","38.53521126760563","48.31788079470199","32.112676056338024","0.0","37.46478873239436","37.2027972027972","16.056338028169012","46.12413793103449","36.394366197183096","42.56","39.05555555555556"
"40.0","25.69014084507042","42.27814569536424","31.26241134751773","37.46478873239436","0.0","25.51048951048951","28.02836879432624","39.834482758620695","25.69014084507042","39.52","22.166666666666668"
"43.0","24.447552447552447","42.27814569536424","25.51048951048951","37.2027972027972","25.51048951048951","0.0","30.825174825174823","37.73793103448276","23.384615384615383","42.56","32.72222222222222"
"44.0","33.183098591549296","48.31788079470199","24.79432624113475","16.056338028169012","28.02836879432624","30.825174825174823","0.0","42.97931034482759","27.830985915492956","41.54666666666667","32.72222222222222"
"45.0","39.834482758620695","49.32450331125828","45.07586206896552","46.12413793103449","39.834482758620695","37.73793103448276","42.97931034482759","0.0","44.02758620689656","42.56","38.786206896551725"
"43.0","21.408450704225352","42.27814569536424","28.901408450704224","36.394366197183096","25.69014084507042","23.384615384615383","27.830985915492956","44.02758620689656","0.0","44.58666666666667","29.555555555555557"
"44.0","41.54666666666667","47.311258278145694","41.54666666666667","42.56","39.52","42.56","41.54666666666667","42.56","44.58666666666667","0.0","38.50666666666667"
"34.0","24.27777777777778","38.25165562913907","34.833333333333336","39.05555555555556","22.166666666666668","32.72222222222222","32.72222222222222","38.786206896551725","29.555555555555557","38.50666666666667","0.0"
