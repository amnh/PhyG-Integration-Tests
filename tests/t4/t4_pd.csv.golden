Alentus,Amblypygid,Aportus,Centruoides,Hadrurus,Limulus,Mastigoproctus,Pauroctonus,Rhiphicephalus,Thelichoris,Thermobius,Vonones
"0.0","35.0","12.0","46.0","45.0","40.0","43.0","44.0","45.0","43.0","44.0","34.0"
"35.0","0.0","37.0","29.0","36.0","24.0","23.0","31.0","38.0","20.0","41.0","23.0"
"12.0","37.0","0.0","46.0","48.0","42.0","42.0","48.0","49.0","42.0","47.0","38.0"
"46.0","29.0","46.0","0.0","30.0","29.0","24.0","23.0","43.0","27.0","41.0","33.0"
"45.0","36.0","48.0","30.0","0.0","35.0","35.0","15.0","44.0","34.0","42.0","37.0"
"40.0","24.0","42.0","29.0","35.0","0.0","24.0","26.0","38.0","24.0","39.0","21.0"
"43.0","23.0","42.0","24.0","35.0","24.0","0.0","29.0","36.0","22.0","42.0","31.0"
"44.0","31.0","48.0","23.0","15.0","26.0","29.0","0.0","41.0","26.0","41.0","31.0"
"45.0","38.0","49.0","43.0","44.0","38.0","36.0","41.0","0.0","42.0","42.0","37.0"
"43.0","20.0","42.0","27.0","34.0","24.0","22.0","26.0","42.0","0.0","44.0","28.0"
"44.0","41.0","47.0","41.0","42.0","39.0","42.0","41.0","42.0","44.0","0.0","38.0"
"34.0","23.0","38.0","33.0","37.0","21.0","31.0","31.0","37.0","28.0","38.0","0.0"
